bpf/tc/attach.go

```diff
@@ -512,3 +537,9 @@ func (ap *AttachPoint) JumpMapFDMapKey() string {
 func (ap *AttachPoint) IfaceName() string {
 	return ap.Iface
 }
+
+func GetTCOpts(ifaceName, hook string) bool {
+	key := ifaceName + "_" + hook
```

Comment:
even if just for testing now, might be worth using the RLock to prevent future burns if anybody uses it for non-test code :sweat_smile: